yarn.lock

```diff
@@ -25472,11 +25472,6 @@ whatwg-fetch@^3.0.0, whatwg-fetch@^3.5.0:
   resolved "https://registry.yarnpkg.com/whatwg-fetch/-/whatwg-fetch-3.5.0.tgz#605a2cd0a7146e5db141e29d1c62ab84c0c4c868"
   integrity sha512-jXkLtsR42xhXg7akoDKvKWE40eJeI+2KZqcp2h3NsOrRnDvtWX36KcKl30dy+hxECivdk2BVUHVNrPtoMBUx6A==
 
-whatwg-fetch@^3.4.1:
```

Comment:
Could you please rebase on `develop`?
Actual diff has been handled via separate Pull Request (#8004).

Reply:
done